pallets/nfts/src/features/approvals.rs

```diff
@@ -76,12 +77,11 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		Self::deposit_event(Event::TransferApproved {
 			collection,
-			item,
+			item: Some(item),
```

Comment:
Is this event somewhere tested?

Reply:
Resolved by adding check for `approve_transfer_works`